O/OpenBLAS/common.jl

```diff
@@ -134,11 +134,18 @@ function openblas_script(;num_64bit_threads::Integer=32, openblas32::Bool=false,
         done
     done
 
+    PATCHELF_FLAGS=()
+
+    # ppc64le and aarch64 have 64KB page sizes, don't muck up the ELF section load alignment
+    if [[ ${target} == aarch64-* || ${target} == powerpc64le-* ]]; then
+        PATCHELF_FLAGS+=(--page-size 65536)
+    fi
```

Comment:
Should we do this in a `patchelf` wrapper?

Reply:
Probably, it would reduce the amount some-one could shoot themselves in the foot.

Reply:
Yes, agreed.